sdk/modules/ui/cpp/include/metavision/sdk/ui/utils/opengl_api.h

```diff
@@ -14,18 +14,29 @@
 
 #ifdef _USE_OPENGL_ES3_
 #include <GLES3/gl3.h>
-// While we keep support for OpenGL, we need to provide a
-// dummy implementation for Glew init function
-#define GLEW_OK 0
-inline int glewInit(void) {
-    return GLEW_OK;
-}
 #else // OpenGL
+#if defined(__APPLE__) && !defined(__linux__)
```

Comment:
I'm not very familiar with MacOS environments, are there cases where both `__APPLE__` and `__linux__` are defined?

Reply:
Yes, I came across this issue during development. This helps anyone that is doing containerized development work. It allows the intended code paths when doing local development of linux containers on Apple silicon.